anonymizer/base.py

```diff
@@ -247,16 +249,33 @@ class Anonymizer(object):
 
     faker = DjangoFaker()
 
+    def __init__(self):
+        super(Anonymizer, self).__init__()
+
+        assert self.attributes is not None, '"attributes" attribute must be set'
+        assert self.model is not None, '"model" attribute must be set'
+
+        self.replacers = []
+        for attname, replacer in self.attributes:
+            if replacer == 'SKIP':
+                continue
+
+            if isinstance(replacer, six.string_types):
+                # 'email' is shortcut for: replacers.email
+                replacer = getattr(replacers, replacer)
+            elif not callable(replacer):
+                raise Exception("Expected callable or string to be passed, got %r." % replacer)
+
+            field = self.model._meta.get_field(attname)
+            self.replacers.append((attname, field, replacer))
```

Comment:
Inspiration taken from the lovely @clarkduvall and Serpy. Trying to do less work in the `alter_object` loop.